entrypoint.sh

```diff
@@ -286,14 +312,46 @@ function seiso_lint() {
 			cat "/opt/goat/log/${pids[$p]}.log"
 			linter_failures+=("${pids[$p]}")
 		else
-			if [[ ${AUTO_FIX:-} == "true" ]]; then
-				cat "/opt/goat/log/${pids[$p]}.log"
-			fi
 			linter_successes+=("${pids[$p]}")
 		fi
 	done
 }
 
+function rerun_lint() {
```

Comment:
Once I strip out the lint functions to their core functionality and learn to pass variables back and forth a little better, we'll have a single lint function.